quark/cli.py

```diff
@@ -124,8 +124,9 @@
 @click.option(
     "--multi-process",
     "num_of_process",
-    type=click.IntRange(min=1),
-    help="Allow analyzing APK with N processes",
+    type=click.INT,
```

Comment:
What's the reason for removing the boundary check?

Reply:
that's a mistake, will fix it.